Druid-Tests/DRIRPrinterTest.class.st

```diff
@@ -0,0 +1,75 @@
+Class {
+	#name : 'DRIRPrinterTest',
+	#superclass : 'DRIRTest',
+	#category : 'Druid-Tests-IR',
+	#package : 'Druid-Tests',
+	#tag : 'IR'
+}
+
+{ #category : 'tests' }
+DRIRPrinterTest >> testPrintConditionalCFG [
+
+	| printer cfg t0 instructionFactory condition |
+	cfg := self setUpCFGWithConditionalWithPhi.
+
+	instructionFactory := cfg instructionFactory.
+	condition := instructionFactory less: 1 than: 2.
+	cfg b1 addInstructionFirst: condition.
+	cfg b1 lastInstruction operands: {
+			DREqualsThanComparison new.
+			condition.
+			true asDRValue }.
+	cfg validate.
```

Comment:
If we want to check this, we should do it at the end before printing (I think). 
We can have a method `self print: cfg` to check and use the printer ;)

Reply:
oops was left from debugging, doesn't make much sense there tbh